### ISSUE TITLE
Simple DNSCrypt needs specify working directory to run

### ISSUE THREAD

**Issue by dimsdein** [bitbeans/SimpleDnsCrypt#449]
I wanted to run GUI on user logon but has issues with locating files. Adding manually an entry to full executable path in registry ("HKCU\Software\Microsoft\CurrentVersion\Run") can not launch it properly even with double clicking the file. Using start command and specifying working dir like `start "Simple DNSCrypt GUI" /D "C:\Program Files\bitbeans\Simple DNSCrypt x64\" "C:\Program Files\bitbeans\Simple DNSCrypt x64\SimpleDnsCrypt.exe"` neither does work.

**Comment by bitbeans** [comment]
```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Run]
"SimpleDNSCrypt"="\"C:\\Program Files\\bitbeans\\Simple DNSCrypt x64\\SimpleDnsCrypt.exe\""
```

**Comment by bitbeans** [comment]
will be fixed in 0.6.9
